apps/nextjs/src/pages/index.tsx

```diff
@@ -31,7 +31,7 @@ export default function Home() {
         <RamRaghavSharma />
         <MinhLe />
         <div>
-          {isSignedIn ? "" : <SignInButton mode="modal" />}
+          {isSignedIn ? "" : <SignInButton />}
```

Comment:
I'm assuming that the `SignInButton` uses the env variable to redirect user to the correct page?

Reply:
Yes